wordpress.php

```diff
@@ -38,6 +38,19 @@
     run('curl {{ url }}/kinsta-clear-cache-all/');
 });
 
+task('wordpress:set-wp-home', function () {
+    // add WP_HOME to environment file
+    $file = realpath($_SERVER['DOCUMENT_ROOT']) . '/config/environments/' . currentHost()->getAlias() . '.php';
+    $lines = file($file);
+    $newLines = [];
+    foreach ($lines as $line) {
+        if (strpos($line, "Config::define('WP_HOME'") === false) {
+            $newLines[] = $line;
+        }
+    }
+    $newLines[] = 'Config::define(\'WP_HOME\', \'' . get('url', 'https://') . '\');';
+    file_put_contents($file, implode('', $newLines));
+});
```

Comment:
Wouldn’t this run on the remote host? it needs to run locally so you can commit it to git (or else a deploy would wipe the change).

`current_path` would hold the path https://github.com/deployphp/deployer/blob/fb6f2154ef36f03e4e4b0644206b06722ee09fd8/recipe/common.php#L99.

you can use `__DIR__` to get root of the project, you can confirm it works by looking at how the rsync works https://github.com/deployphp/deployer/blob/master/contrib/rsync.php#L135

note that no production.php config exists on sites currently, you should check it exists and is writable: `is_writable()`